package.json

```diff
@@ -87,19 +87,19 @@
     "@shapeshiftoss/caip": "workspace:^",
     "@shapeshiftoss/chain-adapters": "workspace:^",
     "@shapeshiftoss/errors": "workspace:^",
-    "@shapeshiftoss/hdwallet-coinbase": "1.52.14",
-    "@shapeshiftoss/hdwallet-core": "1.52.14",
-    "@shapeshiftoss/hdwallet-keepkey": "1.52.14",
-    "@shapeshiftoss/hdwallet-keepkey-webusb": "1.52.14",
-    "@shapeshiftoss/hdwallet-keplr": "1.52.14",
-    "@shapeshiftoss/hdwallet-ledger": "1.52.14",
-    "@shapeshiftoss/hdwallet-ledger-webusb": "1.52.14",
-    "@shapeshiftoss/hdwallet-metamask": "1.52.14",
-    "@shapeshiftoss/hdwallet-native": "1.52.14",
-    "@shapeshiftoss/hdwallet-native-vault": "1.52.14",
-    "@shapeshiftoss/hdwallet-shapeshift-multichain": "1.52.14",
-    "@shapeshiftoss/hdwallet-walletconnectv2": "1.52.14",
-    "@shapeshiftoss/hdwallet-xdefi": "1.52.14",
+    "@shapeshiftoss/hdwallet-coinbase": "1.52.15-alpha.7",
```

Comment:
Install actual 1.52.15 when published

Reply:
Actioned in https://github.com/shapeshift/web/pull/5896/commits/cf7d5c99a8aa0537ef7a53cf4a905913af086498.